doc/user_guide/basics.rst

```diff
@@ -23,10 +23,16 @@ Get a Bucket reference
 
 
 Bucket class
--------------
+------------
 A Bucket contains a set of files which may be restricted, depending on the credentials of the requester.
-Using :ref:`exasol.bucketfs.Bucket <api:exasol.bucketfs.Bucket>` class the user can interact (download, upload, list and delete) files.
```

Comment:
We have an API reference for the Bucket class, but maybe we should now reference the BucketLike, because its a common interface for all Bucket classes, and it is most thoroughly documented.